frontend/packages/kubevirt-plugin/src/components/vm-disks/disk-summary.tsx

```diff
@@ -9,45 +9,31 @@ import { WINTOOLS_CONTAINER_NAMES } from '../modals/cdrom-vm-modal/constants';
 import { VMKind } from '../../types';
 import { V1Disk } from '../../types/vm/disk/V1Disk';
 
-const DiskSummaryRow: React.FC<DiskSummaryRowProps> = ({ title, value }) => (
-  <div className="kubevirt-disk-summary__disk">
-    <div id={`kubevirt-disk-summary-disk-title-${title}`}>{title}</div>
-    <div id={`kubevirt-disk-summary-disk-value-${value}`}>{value}</div>
-  </div>
-);
-
 export const DiskSummary: React.FC<DiskSummaryProps> = ({ disks, vm }) => (
   <div className="kubevirt-disk-summary">
-    {disks.map(({ name }, i) => {
+    {disks.map(({ name }) => {
       const container = getContainerImageByDisk(vm, name);
       const pvc = getPVCSourceByDisk(vm, name);
       const url = getURLSourceByDisk(vm, name);
+      const key = `kubevirt-disk-summary-${name}`;
+      let value = '';
 
       if (_.includes(WINTOOLS_CONTAINER_NAMES, container)) {
-        return (
-          <DiskSummaryRow
-            key={`disk-summary-${name}`}
-            title={`Drive ${i + 1}: Windows Tools`}
-            value={container}
-          />
-        );
-      }
-      if (container) {
-        return (
-          <DiskSummaryRow
-            key={`disk-summary-${name}`}
-            title={`Drive ${i + 1}: Container`}
-            value={container}
-          />
-        );
-      }
-      if (url) {
-        return (
-          <DiskSummaryRow key={`disk-summary-${name}`} title={`Drive ${i + 1}: URL`} value={url} />
-        );
+        value = `Windows Tools: ${container}`;
+      } else if (container) {
+        value = `Container: ${container}`;
+      } else if (url) {
+        value = `URL: ${url}`;
+      } else if (pvc) {
+        value = `PVC: ${pvc}`;
       }
+
       return (
-        <DiskSummaryRow key={`disk-summary-${name}`} title={`Drive ${i + 1}: PVC`} value={pvc} />
+        <div id={key} key={key}>
```

Comment:
Can we change this structure to be 
```
<dl class="oc-vm-details__datalist">
<dt>Windows guest tools</dt>
<dd>Windows Tools: kubevirt/virtio-container-disk</dd>
</dl>
```
It's more semantic and allows us to pull out that `<br />`
We may need to add some styles to this to get it to right when a second `dt`+`dd` is added.

Reply:
makes sense to me, @glekner @suomiy I like this change but I do not know if we shold do it in a folowup PR or in this one, WDYT ?

Reply:
+1, let's do it here

Reply:
unless it already merged :)

Reply:
ok. I'd suggest
```
.co-vm-details-cd-roms--datalist--dd {
margin-bottom: var(--pf-global--spacer--sm);
}

Reply:
> make the new dl a direct child of the dd

I think you meant replacing `kubevirt-disk-summary` with `dl` right?

not sure the PR is in correct state after recent push?

@matthewcarleton can you please take a look again?

Reply:
thanks @suomiy 
ya If we can drop the wrapping `div.kubevirt-disk-summary` and just use a `dl` that would be perfect. We'd also want to add the class name update so we can adjust the margin that currently gets applied to any `dd` in this view.

Reply:
@suomiy @matthewcarleton updated while :sleeping: , also updated the screen shot in the description.

Reply:
one last nitpic...for the morning!
I think we can drop the styling you've added for the `dt` it works pretty well with the default bold text to add some hierarchy.
<img width="337" alt="Screen Shot 2019-12-04 at 3 36 37 PM" src="https://user-images.githubusercontent.com/287880/70174929-32c24580-16ac-11ea-98f3-615a68c42aeb.png">

Reply:
done

Reply:
done